### DISCUSSION TITLE
Duo SAML Support

### DISCUSSION THREAD

**Discussion by rangemab** [snipe/snipe-it#13579] [Q&A] [answered]
I ran across this thread from 2017 (https://github.com/snipe/snipe-it/issues/3420) that says Duo SAML is not supported with Snipe.  Is that still the case in 2023?  We would like to use Duo 2FA in our company's implementation.

**Comment by m4zl** [comment, marked as answer]
I think this can be closed as we successfully implemented Cisco Duo via SAML into our Snipe-IT instance.
